src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -154,6 +165,10 @@ public function startTest(\PHPUnit_Framework_Test $test)
         }
     }
 
+    /**
+     * @param \PHPUnit_Framework_Test $test
+     * @param $time
```

Comment:
This one is wrong as there is no type hint. As a general rule, we never add a `@param` if it does not add value. So, a type hint or a comment if the variable is not clear enough.